frontstage/views/register.py

```diff
@@ -198,7 +201,7 @@ def register_enter_your_details():
 
     # Ensure we have got a valid enrolment code, otherwise go to the sign in page
     if not decrypted_enrolment_code or not validate_enrolment_code(decrypted_enrolment_code):
-        logger.error('Enter Account Details page - invalid enrolment code: ' + str(decrypted_enrolment_code))
+        logger.error('"event" : "invalid code", "error" : "Enter Account Details page - invalid enrolment code: "' + str(decrypted_enrolment_code))
```

Comment:
Don't need event and error here, 
`'"event": "Invalid enrolment code", "Iac_code": "{}"'.format(str(decrypted_enrolment_code))